droid-results/src/main/java/uk/gov/nationalarchives/droid/profile/CsvItemWriter.java

```diff
@@ -220,7 +256,43 @@ public void open(final Writer writer) {
         if (headers == null) {
             headers = Arrays.copyOf(HEADERS, HEADERS.length) ;
         }
-        String[] headersToWrite = getHeadersToWrite(headers);
+    }
+
+    private void writeHeadersForOneLinePerFormatExport(List<? extends ProfileResourceNode> nodes) {
+
+        if (options == ExportOptions.ONE_ROW_PER_FORMAT) {
```

Comment:
Is there a better way to write this check that would extend to cover additional options in future? I can see that `if (options != ExportOptions.ONE_ROW_PER_FILE)` wouldn't work at the moment as it's possible for no option to be set. Could the switch on line 176 be refactored to set the option `ONE_ROW_PER_FILE` if the default case occurs?

Reply:
Actually its a good point, I had specifically moved the option setting line in "ExportTask" so the negative condition would be better